### ISSUE TITLE
can i use transfer learning to apply the pretrained model to a regression task?

### ISSUE THREAD

**Issue by xiao666** [bfelbo/DeepMoji#16]
hello, my data is composed in the form: (a sentence) + (each sentence has a sentiment score in range[-1,1]).
I have a look at the "finetune_youtube_last.py", the parameter of the deepmoji_transfer is number of classes, so does the deepmoji supports regression tasks? 
Thank you!

**Comment by bfelbo** [comment]
No examples should how to do a regression task. It's easy to do through. You could do one of following:

1. Group your range from [-1, 1] into a few intervals (e.g. 5) and then predict them.
2. Modify the final layer of the architecture (in `model_def.py`. For your case it simply involves changing the final activation to a tanh function instead of the softmax function.
